sdk/remoterendering/mixedreality-remoterendering/src/lro/assetConversionPoller.ts

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PollOperationState, Poller, PollOperation } from "@azure/core-lro";
+import { KnownAssetConversionStatus } from "../generated/models/index";
+import { RemoteRendering } from "../generated/operations";
+import { getConversionInternal } from "../internal/commonQueries";
+import { AbortSignalLike } from "@azure/abort-controller";
+import { delay } from "@azure/core-util";
+import { AssetConversion } from "../internal/assetConversion";
+
+export interface AssetConversionPollerOptions {
+  intervalInMs?: number;
+}
+
+export interface AssetConversionOperationState extends PollOperationState<AssetConversion> {
+  /**
+   * The latest response when querying the service. The conversion may or may not be completed.
+   */
+  latestResponse: AssetConversion;
+}
+
+export class AssetConversionOperationStateImpl implements AssetConversionOperationState {
+  latestResponse: AssetConversion;
+
+  constructor(conversionState: AssetConversion) {
+    this.latestResponse = conversionState;
+  }
+
+  get isStarted(): boolean {
+    // We do not take the KnownAssetConversionStatus.NotStarted status into account here.
```

Comment:
The comment was meant to be a implementation note, in case someone (including me in the future) wondered why we didn't map that service value to "hasn't started". I see how it's confusing, so I'll simplify the comment.

I used getters so they could satisfy the PollOperationState interface.